syft/serde/__init__.py

```diff
@@ -1,6 +1,7 @@
 from syft.serde.serde import *
-from syft.serde.torch_serde import *
-from syft.serde.serde import _simplify
-from syft.serde.serde import _detail
-from syft.serde.compression import _compress
-from syft.serde.compression import _decompress
+from syft.serde.msgpack.torch_serde import *
```

Comment:
I don't think is needed any more, since it's imported in `msgpack/__init__.py` below.